pkg/cosign/test_util.go

```diff
@@ -0,0 +1,57 @@
+package cosign
```

Comment:
Update: moving the file to the `pkg/cosign/testing` folder creates an import cycle. Therefore, I'm renaming the file to `helpers_test.go` as you suggested.

Reply:
Update 2: I had to externalize the constants into a new file `pkg/cosign/util/constants.go` to prevent a test import cycle while also allowing the cosign end-to-end tests to leverage that. Let me know if this doesn't work.